dts/arm/nordic/nrf54l15_cpuapp_peripherals.dtsi

```diff
@@ -0,0 +1,410 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ dppic00: dppic@42000 {
+	compatible = "nordic,nrf-dppic";
+	reg = <0x42000 0x808>;
+	status = "disabled";
+};
+
+ spi00: spi@4a000 {
+	/*
+	 * This spi node can be either SPIM or SPIS,
+	 * for the user to pick:
+	 * compatible = "nordic,nrf-spim" or
+	 *              "nordic,nrf-spis".
+	 */
+	compatible = "nordic,nrf-spim";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	reg = <0x4a000 0x1000>;
+	interrupts = <74 NRF_DEFAULT_IRQ_PRIORITY>;
+	max-frequency = <DT_FREQ_M(32)>;
+	easydma-maxcnt-bits = <16>;
+	status = "disabled";
+};
+
+uart00: uart@4a000 {
+	compatible = "nordic,nrf-uarte";
+	reg = <0x4a000 0x1000>;
+	interrupts = <74 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+gpio2: gpio@50400 {
+	compatible = "nordic,nrf-gpio";
+	gpio-controller;
+	reg = <0x50400 0x300>;
+	#gpio-cells = <2>;
+	ngpios = <11>;
+	status = "disabled";
+	port = <2>;
+};
+
+timer00: timer@55000 {
+	compatible = "nordic,nrf-timer";
+	status = "disabled";
+	reg = <0x55000 0x1000>;
+	cc-num = <6>;
+	max-bit-width = <32>;
+	interrupts = <85 NRF_DEFAULT_IRQ_PRIORITY>;
+	prescaler = <0>;
+};
+
+dppic10: dppic@82000 {
+	compatible = "nordic,nrf-dppic";
+	reg = <0x82000 0x808>;
+	status = "disabled";
+};
+
+timer10: timer@85000 {
+	compatible = "nordic,nrf-timer";
+	status = "disabled";
+	reg = <0x85000 0x1000>;
+	cc-num = <8>;
+	max-bit-width = <32>;
+	interrupts = <133 NRF_DEFAULT_IRQ_PRIORITY>;
+	prescaler = <0>;
+};
+
+egu10: egu@87000 {
+	compatible = "nordic,nrf-egu";
+	reg = <0x87000 0x1000>;
+	interrupts = <135 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+radio: radio@8a000 {
+	compatible = "nordic,nrf-radio";
+	reg = <0x8a000 0x1000>;
+	interrupts = <138 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "okay";
```

Comment:
Why? It is enabled for other platforms.

Reply:
I personally think that all peripherals should be disabled by default for soc. Some can be enabled on board level for DK but as few as possible.
When you create an application you have to manually disable everything that was enabled excessively.

You can get away with radio only because our socs are used mainly for radio.

Reply:
Ok, in that case, I will change it to `disabled` for all peripherals. The only ones enabled now are `radio`, `clock`, and `temp`.

Reply:
What I meant was that the only ones, that I changed from enabled to disabled were `radio`, `clock`, and `temp`. The rest was already disabled. Sorry, I only now realized that my wording was not very well.
I also checked and I have not found anything that is passive and could be enabled, although I might be wrong.